code/game/objects/items/food/meatdish.dm

```diff
@@ -226,8 +226,9 @@
 		/datum/reagent/consumable/nutriment/vitamin = 5,
 		/datum/reagent/consumable/nutriment/protein = 2,
 	)
-	tastes = list("rice and meat" = 4, "lettuce" = 2, "soy sauce" = 2)
 	foodtypes = SEAFOOD | MEAT | VEGETABLES
+	tastes = list("rice and meat" = 4, "lettuce" = 2, "soy sauce" = 2)
+	trash_type = /obj/item/reagent_containers/cup/bowl
```

Comment:
Also I reordered tastes to come after foodtypes to match the order in the /obj/item/food def